src/ProxyInterfaceSourceGenerator/FileGenerators/ExtraFilesGenerator.cs

```diff
@@ -40,11 +41,16 @@ public ProxyAttribute(Type type, ProxyClassAccessibility accessibility) : this(t
         {{
         }}
 
-        public ProxyAttribute(Type type, bool proxyBaseClasses, ProxyClassAccessibility accessibility)
+        public ProxyAttribute(Type type, bool proxyBaseClasses, ProxyClassAccessibility accessibility) : this(type, proxyBaseClasses, accessibility, Array.Empty<string>())
+        {{
+        }}
+
+        public ProxyAttribute(Type type, bool proxyBaseClasses, ProxyClassAccessibility accessibility, string[] membersToIgnore)
```

Comment:
Add another override so that you can just use it with a type and memberExpressionsToIgnore